packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -88,31 +92,28 @@ export class OrgList implements vscode.Disposable {
         // scratch orgs parented by other (non-default) devHub orgs
         continue;
       }
-      const isExpired =
-        authFields && authFields.expirationDate
-          ? today >= new Date(authFields.expirationDate)
-          : false;
+      const isExpired = authFields?.expirationDate
```

Comment:
@CristiCanizales the OrgAuthorization does have a `isExpired` field, why not use that?

Reply:
True! I tried it elsewhere looking at other thing and it was always unknown so didn't think about using it here, but I just tried it here and it works fine. Good catch, thanks!